src/widgets/label/lv_label.h

```diff
@@ -158,6 +158,19 @@ void lv_label_set_text_selection_end(lv_obj_t * obj, uint32_t index);
  */
 void lv_label_set_recolor(lv_obj_t * obj, bool en);
 
+#if LV_USE_TRANSLATION
+
+/**
+ * Bind a translation tag to this label. Memory will be allocated to store the tag by the label.
+ * The label text will automatically update when the language changes
```

Comment:
```suggestion 
 * Set a translation tag for this label. Memory will be allocated to store the tag by the label.
 * The label text will automatically update when the language is changed by `lv_translation_set_language`.
```

Bind is kind of reserved word for data bindings. Let's use something else to avoid confusion. 
```suggestion
```